posts/ja/2023-02-18-23.0.0.1_ja.adoc

```diff
@@ -0,0 +1,199 @@
+
+In link:{url-about}[Open Liberty] 23.0.0.1:
+
+* <<eplv2, Open LibertyのライセンスはEPLv2になりました>>
+* <<bugs, 主なバグフィックス>>
+
+link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23001+label%3A%22release+bug%22[23.0.0.1].で修正されたバグの一覧を見ることができます。
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
+
+
+[#run]
+
+== 23.0.0.1を使用してアプリを実行する
+
+link:{url-prefix}/guides/maven-intro.html[Maven]を使う時には下記の設定を使ってください。
+
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>23.0.0.1</version>
+    <type>zip</type>
+</dependency>
+----
+
+link:{url-prefix}/guides/gradle-intro.html[Gradle]の場合は、下記の設定を使ってください。
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[23.0.0.1,)'
+}
+----
+
+Dockerを使っているときは、下記のイメージを使ってください。
+
+[source]
+----
+FROM open-liberty
+----
+
+または、Open Libertyの link:{url-prefix}/downloads/[ダウンロードページ]からダウンロードしてください。
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack_ja.svg[Ask a question on Stack Overflow, align="center"]
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/24069
+// Contact/Reviewer: ReeceNana,cbridgha
+// // // // // // // // 
+[#eplv2]
+== Open LibertyのライセンスはEPLv2になりました
+2022年12月7日、ink:{url-prefix}/blog/2022/12/07/relicense-eplv2-proposal.html[Open LibertyのEPLv2への再ライセンス提案]のブログ記事を公開し、Open LibertyのライセンスをEPLv1からEPLv2への変更についてコミュニティに意見を求めました。 この提案に対して否定的なフィードバックがなかったため、私たちはこの変更を進めました。結果として、Open Liberty 23.0.0.1はEPLv2ライセンスのもとでリリースされます。 EPLv1からEPLv2への移行の理由と影響についてのより詳しい情報は、link:{url-prefix}/blog/2022/12/07/relicense-eplv2-proposal.html[proposal blog記事]をご覧ください。
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+[#bugs]
+== 主なバグ修正
+
+以下のセクションでは、このリリースで修正したバグの一部について説明します。興味がある場合はlink:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23001+label%3A%22release+bug%22[full list of bugs fixed in 23.0.0.1]を参照してください。
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23885[JDK20 以降では、最低でも 1.8 の jdkSourceLevel を使用]
++
+JDK 20 では、`javac` が Java 1.7 のコンパイルのサポートを終了しています。この変更により、JSPで `jdkSourceLevel="17"` (Java version 1.7) を指定した場合、以下のエラーが発生します。
++
+[source]
+----
+com.ibm.ws.jsp.JspCoreException: JSPG0049E: /xxxxxxxxx.jsp failed to compile :
+error: Source option 7 is no longer supported. Use 8 or later.
+----
++
+Java 20以降をランタイムとする場合、ユーザが指定した `jdkSourceLevel` に低いレベルが指定されると、 `18` に上書きされます。
+修正後、Java 20以降を使用する場合、`jdkSourceLevel` に `17` 以下が指定されると、その値は `18` に上書きされるようになりました。
+
+* link:https://github.com/OpenLiberty/open-liberty/issues/23883[ファイルモニタリングでデフォルトのkeystore ファイルが検出されない]
++
+`updateTrigger` 属性に `mbean` (デフォルト値) を使用すると、変更が発生したときにデフォルトの keystore ファイルがリロードされないという問題が発生しました。
```

Comment:
Unsure if `keystore` has an appropriate translate or if it should remain in English

Reply:
Same as above.